### ISSUE TITLE
Uploading a new dataset has a 'N files missing' message

### ISSUE THREAD

**Issue by spikelynch** [Weed-AI/Weed-AI#664]
ie if I am creating a new dataset and upload a weedcoco.json with four image references, the upload-images step says '4 images missing' - this should only appear if I'm editing an existing dataset

**Comment by ElevnLi** [comment]
I think we can just rephrase it to "4 images to be uploaded", which gives a sense of how many image are expected.

**Comment by spikelynch** [comment]
yep sounds good
